### PR TITLE
support Kotlin/Wasm for apollo-adapters

### PR DESCRIPTION
**Pull request by joreilly**
addresses https://github.com/apollographql/apollo-kotlin/issues/5795

### CODE REVIEW

**Review by martinbonnin — COMMENTED** [review]
Nice!

File: libraries/apollo-adapters/src/wasmJsTest/kotlin/com/apollographql/apollo3/adapter/test/BigDecimalTests.kt [hunk 1]
```diff
@@ -0,0 +1,72 @@
+package com.apollographql.apollo3.adapter.test
+
+import com.apollographql.apollo3.adapter.BigDecimal
+import kotlin.test.*
+
+class BigDecimalTests {
```
[the remark this answers is not in the document]
Comment (joreilly) [reply libraries/apollo-adapters/src/wasmJsTest/kotlin/com/apollographql/apollo3/adapter/test/BigDecimalTests.kt:6]: yeah, was wondering about that too (they are identical)

[the remark this answers is not in the document]
Comment (martinbonnin) [reply libraries/apollo-adapters/src/wasmJsTest/kotlin/com/apollographql/apollo3/adapter/test/BigDecimalTests.kt:6]: Let me try. Also, I wonder why they are `jsTest` and not `commonTest`, I'll dig

[the remark this answers is not in the document]
Comment (martinbonnin) [reply libraries/apollo-adapters/src/wasmJsTest/kotlin/com/apollographql/apollo3/adapter/test/BigDecimalTests.kt]: I just pushed a commit to [share](https://github.com/apollographql/apollo-kotlin/pull/5803/commits/a482b0d084e378fcc3f6655bcbd8a7ef034e2a4a) the tests between JS and Wasm.

I tried moving to `commonTest`. Unfortunately, we still have behavioural discrepencies there :/...We should probably use a proper BigDecimal lib. Anyone with recommendations?

**Comment by martinbonnin** [comment]
Thanks!

**Pull request merged**